### PR TITLE
[DCJ-71] Create page to edit DAC

### PR DESCRIPTION
**Pull request by aarohinadkarni**
### Addresses
https://broadworkbench.atlassian.net/browse/DCJ-71

### Summary
Takes content of the Edit DAC action on the DAC Chair Console into a new page for each DAC found at a new `/manage_edit_dac/{dacId}` route.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by fboulnois — APPROVED** [review]
looks great 👍

File: src/pages/manage_dac/ManageEditDac.jsx [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+    fetchData();
+    setIsLoading(false);
+  }, [dacId, setState]);
+
+  const okHandler = async (event) => {
+    event.preventDefault();
+
+    let currentDac = state.dac;
+    if (state.dirtyFlag) {
+      // TODO: removed the admin check .. not sure how to get the user role if not passed in as prop
```
Comment (okotsopoulos) [inline src/pages/manage_dac/ManageEditDac.jsx]: Is this TODO actionable or meant to be ticketed, or is it serving as a FYI?

**Review by okotsopoulos — APPROVED** [review]
Looks good!

Is there any support available in this codebase for component tests?  Especially cases where the underlying algorithm is so involved as to benefit from comments (thanks for those!).  I could see those benefitting from repeated verification in tests.

File: src/pages/manage_dac/ManageEditDac.jsx (hunk 1 above)
Comment (aarohinadkarni) [reply src/pages/manage_dac/ManageEditDac.jsx]: It is actionable!! I missed that. Thanks for the call out 😄

**Comment by aarohinadkarni** [comment]
> Is there any support available in this codebase for component tests? Especially cases where the underlying algorithm is so involved as to benefit from comments (thanks for those!). I could see those benefitting from repeated verification in tests. 

@okotsopoulos I'm pretty sure there is support available for component tests! Personally I haven't written any in this repo, but I can investigate.

**Comment by aarohinadkarni** [comment]
> I'm pretty sure there is support available for component tests! Personally I haven't written any in this repo, but I can investigate.

After discussing in standup, I've created this ticket to write a test for this component: https://broadworkbench.atlassian.net/browse/DCJ-405.

**Pull request merged**